src/libs/actions/Report.js

```diff
@@ -1639,15 +1639,21 @@ function removeEmojiReaction(reportID, originalReportAction, emoji) {
  * @returns {Promise}
  */
 function toggleEmojiReaction(reportID, reportAction, emoji, paramSkinTone = preferredSkinTone) {
-    const message = reportAction.message[0];
+    const latestReportAction = ReportActionsUtils.getReportAction(reportID, reportAction.reportActionID);
```

Comment:
I find this variable name to be a little bit misleading. Usually `latestReportAction` means the latest message on a chat, as opposed to a message sent previously. Here we're really accessing the `latestVersionOfTheReportAction`.

Let's make this method consistent by having it take in a `reportActionID` instead of the `reportAction`, which we now only use to access the ID anyways.

Reply:
Updated code for requested changes @neil-marcellini.